lib/travis/api/v3/queries/crons.rb

```diff
@@ -0,0 +1,8 @@
+module Travis::API::V3
+  class Queries::Crons < Query
+
+    def find(branch)
```

Comment:
Should probably move to the Con query. Would expect the Crons query to handle collections of cron jobs.